lib/WP_Auth0_LoginManager.php

```diff
@@ -170,15 +170,15 @@ public function init_auth0() {
       }
     } catch (WP_Auth0_LoginFlowValidationException $e) {
 
-      $msg = __( 'There was a problem with your log in', 'wp-auth0' );
+      $msg = __( 'There was a problem with your log in. ', 'wp-auth0' );
       $msg .= ' '. $e->getMessage();
       $msg .= '<br/><br/>';
       $msg .= '<a href="' . wp_login_url() . '">' . __( '← Login', 'wp-auth0' ) . '</a>';
       wp_die( $msg );
 
     } catch (WP_Auth0_BeforeLoginException $e) {
 
-      $msg = __( 'You have logged in successfully, but there is a problem accessing this site', 'wp-auth0' );
+      $msg = __( 'You have logged in successfully, but there is a problem accessing this site. ', 'wp-auth0' );
```

Comment:
i18n?

Reply:
@lbalmaceda - yes?

Reply:
Should probably change these back as this will break existing i18n for this message as it's changed.

Reply:
It's not that the main message changed but fixed grammatically, I see no need to rollback this change.

Reply:
Well fixing it breaks it for existing use, no huge feeling either way.

Reply:
We don't have a PO file in here that we keep up to date so if someone is generating that and translating for themselves, they'll be used to doing that on each release (or used to things breaking/not translating).